<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DN8 Simulator</string>
  </property>
  <widget class="QProgressBar" name="pgbDrill">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>160</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
    </font>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_DrillTime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>30</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>30</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbStart">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>60</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbStop">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>60</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_CheckTime">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>30</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdUncheckedNum">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>5</number>
   </property>
   <property name="midLineWidth">
    <number>5</number>
   </property>
   <property name="digitCount">
    <number>5</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdCheckedNum">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>5</number>
   </property>
   <property name="midLineWidth">
    <number>5</number>
   </property>
   <property name="digitCount">
    <number>5</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_DrillLoading">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>30</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Stp1Loading">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>30</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Stp2Loading">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>30</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_SideLoading">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>30</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Side_STP1_To_Buff">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>30</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Drill_STP1_To_Buff">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>30</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_STP1Time">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>30</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>30</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Drill_To_STP1">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>30</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Side_To_STP1">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>30</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="pgbSTP1_A">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>160</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
    </font>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="pgbSTP1_B">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>160</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
    </font>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Recent Loading : </string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_RecentJob">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 114, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbSTP1B">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>130</y>
     <width>64</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <item>
    <property name="text">
     <string>공급대기</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>가공대기</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>배출대기</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>가공중</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cmbSTP1A">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>40</y>
     <width>64</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <item>
    <property name="text">
     <string>공급대기</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>가공대기</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>배출대기</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>가공중</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
